packages/ui/src/mocks/data/raw/openings.json

```diff
@@ -36,7 +36,7 @@
     "stakeAmount": 6000,
     "rewardPerBlock": 400,
     "version": 1,
-    "type": "LEAD",
+    "type": "LEADER",
```

Comment:
Please add this to the mocks generators as well.

The `all` option should regenerate all mocks now using proper dependencies.